packages/acceptit-Tests.package/ACStepParsingTest.class/instance/testItParsesTrue.st

```diff
@@ -0,0 +1,11 @@
+as yet unclassified
+testItParsesTrue
+	
+	|ac method|
+	ACReadMeFactory executeReadMe.
+	ac := (Smalltalk classNamed: 'ACCalculatorUserStory') new.
```

Comment:
'ACCalculatorUserStory' should be changed to ACUserStoryFactory userStoryTitleTrimmed. If we build code dependent on the ACReadMe we can use the same methods for getting the constant text as well. When refactoring the readme and the ACUserStoryFactory which we should do by the way, we will be able to see the all dependent functions by checking the senders.